test/albums.test.js

```diff
@@ -119,4 +134,106 @@ describe('albums api tests', () => {
       )
     );
   });
+
+  test('getAlbumsByUser without jwt returns forbidden error', done =>
+    request(server)
+      .get('/users/1/albums')
+      .then(response => {
+        expect(response.body.message).to.equal('jwt must be provided');
+        expect(response.body.internal_code).to.equal('forbidden_error');
+        done();
+      }));
+
+  test('getAlbumsByUser with jwt and regular user and id of another user returns unauthorized error', done =>
+    User.createWithHashedPassword(mockedUser).then(() =>
+      User.createWithHashedPassword(otherUser).then(user =>
+        request(server)
+          .post('/users/sessions')
+          .query({
+            email: user.email,
+            password: 'Wolox1189!'
+          })
+          .then(res =>
+            request(server)
+              .get('/users/1/albums')
+              .set('Authorization', res.body.token)
+              .then(response => {
+                expect(response.body.message).to.equal(
+                  'You must have admin permissions to get the albums of another user'
+                );
+                expect(response.body.internalCode).to.equal('unauthorized_error');
+                done();
+              })
+          )
+      )
+    ));
+
+  test('getAlbumsByUser with jwt and regular user and own id returns his albums', done =>
+    User.createWithHashedPassword(mockedUser).then(user =>
+      request(server)
+        .post('/users/sessions')
+        .query({
+          email: user.email,
+          password: 'Wolox1189!'
+        })
+        .then(res =>
+          request(server)
+            .post('/albums/1')
```

Comment:
when you call this endpoint, you must do the respective mock because in background our api is calling the albums api and if these fail our test will fail.

extra: think in put the mocks in util.js too

Reply:
It's true. Now I do :)